### PR TITLE
Add getting started info to readme

### PR DESCRIPTION
**Pull request by elijahsummers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ EvilPlot is about combinators for graphics.
 
 Documentation for EvilPlot is available [here](https://cibotech.github.com/evilplot).
 
+## Getting Started
+Add EvilPlot to your build with:
+```scala
+resolvers += Resolver.bintrayRepo("cibotech", "public")
+libraryDependencies += "com.cibo" %% "evilplot" % "0.2.0" // Use %%% instead of %% if you're using ScalaJS
```
Comment (zacharygraziano) [inline README.md]: current version is 0.2.1.
I guess we could also just make a note that the current version is displayed at the top on the badge. Or update this on releases.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ EvilPlot is about combinators for graphics.
...
+```
+Take a look at the full [getting started guide](https://cibotech.github.com/evilplot/getting-started.html).
```
Comment (zacharygraziano) [inline README.md]: github.io, right?

File: README.md [hunk 3]
```diff
@@ -7,6 +7,14 @@ EvilPlot is about combinators for graphics.
 
 Documentation for EvilPlot is available [here](https://cibotech.github.com/evilplot).
```
Comment (zacharygraziano) [inline README.md]: .io, right?

**Comment by elijahsummers** [comment]
ah thank you @zacharygraziano! too much copy/paste 🤦‍♂️

**Pull request merged**